projects/seb-ng-wizard/src/lib/left-navigation/left-navigation.component.scss

```
@@ -164,6 +164,12 @@ $transitionTimeFast: 0.1s;
   }
 }
 
+.use-navbar {
+  :host-context {
+    top: 3.5rem;
+  }
+}
+
 @include breakpoint-min(#{map-get($grid-breakpoints, 'md')}) {
```

Bootstrap has a built in mixin for adding media queries which we can leverage like this:
```scss
@include media-breakpoint-up(md) {
   ...
}
```
or
```scss
@include media-breakpoint-down(sm) {
   ...
}
```

```suggestion
@include media-breakpoint-up(md) {
```